### DISCUSSION TITLE
Proper documentation

### DISCUSSION THREAD

**Discussion by ken4ward** [martinpaljak/GlobalPlatformPro#338] [Q&A]
I'm just getting into the JavaCard world, and as much effort that has been put into providing these good libraries, it seems there are no documentation for any. The only place I have seen a documentation is NXP (JCOP). Are there some that I'm unaware of? If not, I'll be interested to be part of the documenters. Please, let me know if I can.

**Comment by martinpaljak** [comment]
Feel free to make PR-s with documentation.
